README.md

```diff
@@ -1,11 +1,22 @@
 # deltachat-node
 
-> node.js bindings for deltachat-core
+> node.js bindings for [`deltachat-core`][deltachat-core]
 
 [![Build Status](https://travis-ci.org/deltachat/deltachat-node.svg?branch=master)](https://travis-ci.org/deltachat/deltachat-node)
+![Node version](https://img.shields.io/node/v/deltachat-node.svg)
```

Comment:
Currently this badge is grayed out. This is because the module hasn't been published yet. As soon as the first version has been published, the badge will be filled in with `10.6.0` (based on the value in `package.json`).